apps/homescreen/js/app.js

```diff
@@ -687,6 +687,11 @@ const SETTINGS_VERSION = 0;
 
           default:
             icon.launch();
+
+            // Launching an app
+            if (icon.app) {
```

Comment:
I notice here also that the performance marker is generated after the icon launch occurs, which is different than the way the current homescreen works. I think in this instance we want the marker before launch so the numbers are consistent and have a common relationship.